src/components/project/InstitutionSearchInput.vue

```diff
@@ -0,0 +1,84 @@
+<script setup lang="ts">
+import { ref } from 'vue'
+const props = defineProps<{
+  search: (text: string) => Promise<any[]>
+  creation: (id: number, name: string) => void
+}>()
+
+const items = ref([])
+let text = ref()
+let institutionId: number = null
+
+async function handleInput(event: any) {
+  const text = event.target.value
+  if (text.length < 3) {
+    items.value = []
+  } else {
+    items.value = await props.search(text)
+    institutionId = null
+  }
+}
+
+function handleCreation(institutionId: number, name: string) {
+  // to handle a case where the user does not decide to use the dropdown select
+  if (institutionId == null) {
+    const institution = items.value.find((i: any) => i.name == name)
+    institutionId = institution ? institution.institution_id : null
+  }
+
+  props.creation(institutionId, name)
+}
+
+function handleSelect(item: any) {
+  institutionId = item.institution_id
+  text.value = item.name
+  items.value = []
+}
+</script>
+<template>
+  <div class="input-group">
+    <input
+      @input="handleInput"
+      v-model="text"
+      class="form-control border"
+      type="search"
+    />
+    <button type="button" @click="handleCreation(institutionId, text)">
+      Save
+    </button>
+    <ul v-if="items.length > 0" class="results">
+      <template v-for="item in items">
+        <li @mousedown="handleSelect(item)">
+          <slot name="item" v-bind="item">{{ item.name }}</slot>
+        </li>
+      </template>
+    </ul>
+  </div>
+</template>
```

Comment:
Rather than create this again, let's use Vue inheritance so that we can decompose searching institutions from saving institutions. We want `InstitutionSearchInput` to only search institutions and  `institutions/CreateView` to add institutions to the project.

For `InstitutionSearchInput` - directly use the `SearchSelectInput` - just like `TaxaSearchInput` and move all searching features to this vue. The `select` event will be emitted to the `CreateView.vue` which you can update a reference.

In `CreateView.vue` that will do all the creation. That's where the 'save' button is located.  Get the institutionId and name from the `InstitutionSearchInput` when the button is clicked.